src/lib/addons/feature-event-formatter-md.ts

```diff
@@ -140,6 +142,18 @@ const EVENT_MAP: Record<string, IEventData> = {
         action: '*{{user}}* sent to review change request {{changeRequest}}',
         path: '/projects/{{event.project}}/change-requests/{{event.data.changeRequestId}}',
     },
+    [CHANGE_REQUEST_SCHEDULED]: {
+        action: '*{{user}}* scheduled change request {{changeRequest}} in project *{{event.project}}*',
+        path: '/projects/{{event.project}}/change-requests/{{event.data.changeRequestId}}',
+    },
+    [CHANGE_REQUEST_SCHEDULED_APPLICATION_SUCCESS]: {
+        action: '*{{user}}* successfully applied scheduled change request {{changeRequest}} in project *{{event.project}}*',
```

Comment:
Who is the `user` in this case? The same as the one that's usually used for CR applications?

And if we're changing the wording of the failed event, we could also change this one a bit:

```suggestion
        action: 'Successfully applied the scheduled change request {{changeRequest}} by *{{user}}* in project *{{project}}*.',
```

Reply:
yes, the 'applier'.